egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -417,15 +442,38 @@ def SmoothProbabilityDistribution(list, smoothing_weight=0.3):
       # Normalize the probability
       sum_p = sum(item.probability for item in list)
       for item in list:
-          item.probability = item.probability / sum_p
+          item.probability = item.probability / sum_p * target_sum
```

Comment:
E.g. the probability of that set is 0.5, then the sum of probabilities of that set should be equal to 0.5